azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeIncomingCallEndEvent.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.models;
+
+/**
+ * Incoming call end event.
+ */
+public class CallCompositeIncomingCallEndEvent {
```

Comment:
Do we need new event, or do we just want to build on CallCompositeCallStateChangedEvent (i.e. by adding CallType to that if necessary?).